src/utils/table.go

```diff
@@ -25,13 +25,22 @@ import (
 	ui "github.com/gizak/termui/v3"
 )
 
+type CustomColColor struct {
+	ColNumber int
+	ColColor  ui.Color
+}
+
```

Comment:
@panzerox123 would it make sense to turn this into a map instead? It would simplify the colour setting within the `Draw()` function too.

Reply:
That makes sense. I'll give it a try